JKA-1587 チャプター共通メソッドの整理（きしもと）

yahuhuhuhuu opened the pull request:
## 概要
Instructor / Manager それぞれの ChapterController に存在していた
重複メソッドを整理し、Policy パターンに統一するための修正を行いました。

- 削除したもの
- Manager側で不要になった機能
InstructorとManegerの両方に存在していたshow、put、bulkDelete、deleteAll、sort 、patchStatus、putStatus

- Manager側に残したもの
Instructor 側には存在していなかったdelete（単体削除）、updateStatus（単体公開/非公開）

## 動作確認手順
- マネージャー権限のある講師でログイン
- チャプター新規作成API
![POSTチャプター新規作成API](https://github.com/user-attachments/assets/27e6a02a-8331-48e2-8396-c37f0afa99d7)

- 選択済チャプターの削除API
![DELETEコース１選択済チャプターの削除API](https://github.com/user-attachments/assets/7af8be96-c588-4c0a-8e96-c106a4fc65a1)
![DELETEコース２選択済チャプターの削除API](https://github.com/user-attachments/assets/086cd16f-bfdd-4948-9372-5a294e715272)

- チャプター並び替えAPI
![POSTチャプター並び替えAPI](https://github.com/user-attachments/assets/1dbf6717-3065-4dd2-9137-712b447cfea2)

- チャプター一括更新API
![PUTチャプター一括更新API](https://github.com/user-attachments/assets/9fb2ddfb-2edb-4fd9-9a87-8d4bd0bea4dc)
![PUTチャプター一括更新API後のDB](https://github.com/user-attachments/assets/69524d95-b23d-48c1-b12e-f12be9e37791)

- チャプターの公開/非公開API
![PATCHチャプターの公開非公開API](https://github.com/user-attachments/assets/257fc676-73e2-49b1-a557-3f34dcfacb82)

- 全チャプター削除API
![DELETEコース２全チャプター削除API](https://github.com/user-attachments/assets/41ecce72-d471-4fb0-8058-f5b23ae1574d)

- チャプター詳細情報を取得
![GETチャプター詳細情報を取得](https://github.com/user-attachments/assets/de02409e-38af-463f-9ede-8eab02d3f8b7)

- チャプター更新API
![PUTチャプター更新APIボディ](https://github.com/user-attachments/assets/5f1790b8-588b-4b7b-b311-7261897bff4b)
![PUTチャプター更新APIパラメータ](https://github.com/user-attachments/assets/6669b4d4-09c9-4888-b299-3e851c428c9c)


## 確認してほしいこと
マネージャーとインストラクターに共通していたルーティングを削除しましたが、問題なく動作していると思います。
実施内容の確認をお願いいたします。

yahuhuhuhuu committed:
JKA-1587 チャプター共通メソッドの整理

kou-tech reviewed (commented):
インポートしなくなったリクエストファイルは他で使わないので、削除お願いします！

yahuhuhuhuu committed:
JKA-1587 チャプター共通メソッドの整理

yahuhuhuhuu commented:
ありがとうございます。削除することを失念していました。
ご意見いただいた通り、Manager側で不要になったbulkDelete、deleteAll、patchStatus、put、putStatus、show、sortRequestを削除しました。

deleted:    app/Http/Requests/Manager/Chapter/BulkDeleteRequest.php
deleted:    app/Http/Requests/Manager/Chapter/DeleteAllRequest.php
deleted:    app/Http/Requests/Manager/Chapter/PatchStatusRequest.php
deleted:    app/Http/Requests/Manager/Chapter/PutRequest.php
deleted:    app/Http/Requests/Manager/Chapter/PutStatusRequest.php
deleted:    app/Http/Requests/Manager/Chapter/ShowRequest.php
deleted:    app/Http/Requests/Manager/Chapter/SortRequest.php

Commit:
自動テストを削除。
